vm_dump.c

```diff
@@ -474,7 +474,7 @@ rb_vmdebug_thread_dump_state(VALUE self)
 # ifdef HAVE_LIBUNWIND
 #  undef backtrace
 #  define backtrace unw_backtrace
-# elif defined(__APPLE__) && defined(__x86_64__) && defined(HAVE_LIBUNWIND_H)
```

Comment:
cc @nobu because you added this `86_64` check in https://github.com/ruby/ruby/commit/5c7f36a1b81216d1a31652c54b628bc364f08245

Do you think this patch is ok?